test/unit/pages/prescription/prescriptionFormConstants.test.js

```diff
@@ -895,12 +1051,48 @@ describe('prescriptionFormConstants', function() {
     ]);
 
     _.each(prescriptionFormConstants.insulinModelOptions, device => {
-      expect(device.value).to.be.a('string');
       expect(device.label).to.be.a('string');
     })
   });
 
-  // it('should export the list of valid country codes', function() {
-  //   expect(prescriptionFormConstants.validCountryCodes).to.be.an('array').and.to.eql([1]);
-  // });
+  it('should export the list the calculator method options', function() {
```

Comment:
```suggestion
  it('should export the list of the calculator method options', function() {
```
It seems this copy/paste happens a few more times below as well

Reply:
Fixed here and in the other occurrences.